Core/GDCore/Project/Project.cpp

```diff
@@ -938,6 +950,8 @@ Project& Project::operator=(const Project& other) {
 
 void Project::Init(const gd::Project& game) {
   name = game.name;
+  categories = game.categories;
+  description: game.description;

```

Comment:
Isn't there a problem here with the `:` ?

Reply:
wow yes. I don't know why I didn't get an error for that